src/styles/App.css

```diff
@@ -0,0 +1,42 @@
+.App {
```

Comment:
How much of this file is lift over boilerplate code? If not being used it should be deleted